outwatch/src/main/scala/outwatch/util/package.scala

```diff
@@ -0,0 +1,13 @@
+package outwatch
+
+import monix.reactive.Observable
+
+package object util {
+  /**
+   * A Function that applies an Action onto the Stores current state.
+   * @param reducer The reducing function
+   * @tparam A The Action Type
+   * @tparam M The Model Type
+   */
+  type Reducer[A, M] = (M, A) => (M, Observable[A])
```

Comment:
Why did we move the `Reducer` out of the `Store` object? To me, it felt right to be namespaced with the Store, because this is not any Reducer, but the Reducer for the Store.